src/dbg/_exports.cpp

```diff
@@ -1272,6 +1273,15 @@ extern "C" DLL_EXPORT duint _dbg_sendmessage(DBGMSG type, void* param1, void* pa
     }
     break;
 
+    case DBG_SELCHANGED:
+    {
+        PLUG_CB_SELCHANGED plugSelChanged;
+        plugSelChanged.hWindow = (int)param1;
+        GuiSelectionGet(GUI_DISASSEMBLY, &plugSelChanged.selection);

```

Comment:
This will take at least 1ms on every selection change. I think it's better to have `DbgSelChanged(int hWindow, duint addr)` and if plugins need the size (almost never) they can query it manually...